internal/core/items.rs

```diff
@@ -961,7 +962,10 @@ pub struct WindowItem {
 }
 
 impl Item for WindowItem {
-    fn init(self: Pin<&Self>, _self_rc: &ItemRc) {}
+    fn init(self: Pin<&Self>, _self_rc: &ItemRc) {
+        #[cfg(feature = "std")]
+        self.full_screen.set(std::env::var("SLINT_FULLSCREEN").is_ok());
```

Comment:
IIRC, this is called after all properties have been initialized, so that'd overwrite the property set in the .slint file
(But I might remember wrong wrong)

Reply:
I feel that I may not have enough knowledge about the Slint internals (yet) :)

From my testing it does not overwrite the full-screen property in the .slint file.

Reply:
Indeed, that seems to be working fine.